rules/windows/process_creation/win_apt_mustangpanda.yml

```diff
@@ -13,15 +13,15 @@ logsource:
     product: windows
 detection:
     selection1:
-        CommandLine: 
-            - '*Temp\wtask.exe /create*'
-            - '*%windir:~-3,1%%PUBLIC:~-9,1%*'
-            - '*/E:vbscript * C:\Users\\*.txt" /F'
-            - '*/tn "Security Script *'
-            - '*%windir:~-1,1%*'
+        CommandLine|contains: 
+            - 'Temp\wtask.exe /create'
+            - '%windir:~-3,1%%PUBLIC:~-9,1%'
+            - '/tn "Security Script '
+            - '%windir:~-1,1%'
+            - '/E:vbscript * C:\Users\\*.txt" /F'
```

Comment:
the asterisks here are going to break the logic %) I suggest the following solution:

```
detection:
    selection1:
        - CommandLine|contains:
            - 'Temp\wtask.exe /create'
            - '%windir:~-3,1%%PUBLIC:~-9,1%'
            - '/tn "Security Script '
            - '%windir:~-1,1%'
        - CommandLine|contains|all:
            - '/E:vbscript'
            - 'C:\Users\'
            - '.txt'
            - '/F'
    selection2:
        Image|endswith: 'Temp\winwsh.exe'
    condition: 1 of them
```

Reply:
Applied